packages/admin-ui/src/app/_services_and_types/participants.service.ts

```diff
@@ -43,6 +43,79 @@ export class ParticipantsService {
 		// this._http.
 	}
 
+	validateSettlementInitiationFile(
```

Comment:
New code we put at the end of the file, unless we're adding private methods which should come before the public.